tests/core/test_model.py

```diff
@@ -290,6 +291,33 @@ def test_model_download(tmpdir, cls, filename):
         assert isinstance(task, cls)
 
 
+class DummyTask(Task):
+    def __init__(self):
+        super().__init__()
+
+        self.model = nn.Sequential(
```

Comment:
very nit: for testing: Can we use another attribute name here (not `self.model`)? This would ease debugging since it is also called `self.model` inside the embedder class.

Reply:
Good call! Done :smiley: